buildSrc/src/main/groovy/io/micronaut/guides/GuideProjectGenerator.groovy

```diff
@@ -85,6 +92,27 @@ class GuideProjectGenerator implements AutoCloseable {
         Map config = new JsonSlurper().parse(configFile) as Map
         boolean publish = config.publish == null ? true : config.publish
 
+        if(publish){
```

Comment:
why only if publish true?

Reply:
because if we validate for base guide (which can have even only 1 property) we can't put any required field in the schema